src/components/utils/popover/popover-desktop.ts

```diff
@@ -162,31 +167,72 @@ export class PopoverDesktop extends PopoverAbstract {
    *
    * @param item – item to show nested popover for
    */
-  protected override showNestedItems(item: PopoverItemDefault): void {
+  protected override showNestedItems(item: WithChildren<PopoverItemDefault> | WithChildren<PopoverItemHtml>): void {
     if (this.nestedPopover !== null && this.nestedPopover !== undefined) {
       return;
     }
     this.showNestedPopoverForItem(item);
   }
 
   /**
-   * Additionaly handles input inside search field.
-   * Updates flipper items considering search query applied.
+   * Handles hover events inside popover items container
    *
-   * @param data - search event data
-   * @param data.query - search query text
-   * @param data.result - search results
+   * @param event - hover event data
    */
-  private handleSearch = (data: { query: string, items: SearchableItem[] }): void => {
-    /** List of elements available for keyboard navigation considering search query applied */
-    const flippableElements = data.query === '' ? this.flippableElements : data.items.map(item => (item as PopoverItem).getElement());
+  protected handleHover(event: Event): void {
+    const item = this.getTargetItem(event);
 
-    if (this.flipper.isActivated) {
-      /** Update flipper items with only visible */
-      this.flipper.deactivate();
-      this.flipper.activate(flippableElements as HTMLElement[]);
+    if (item === undefined) {
+      return;
     }
-  };
+
+    if (this.previouslyHoveredItem === item) {
+      return;
+    }
+
+    this.destroyNestedPopoverIfExists();
+
+    this.previouslyHoveredItem = item;
+
+    if (!item.hasChildren) {
+      return;
+    }
+
+    this.showNestedPopoverForItem(item);
+  }
+
+  /**
+   * Sets CSS variable with position of item near which nested popover should be displayed.
+   * Is used for correct positioning of the nested popover
+   *
+   * @param nestedPopoverEl - nested popover element
+   * @param item – item near which nested popover should be displayed
+   */
+  protected setTriggerItemPositionProperty(nestedPopoverEl: HTMLElement, item: WithChildren<PopoverItemDefault> | WithChildren<PopoverItemHtml>): void {
```

Comment:
Is a `TriggerItem` the same as a `TargetItem` which is used in other places?

Reply:
No. Target item is the item that was clicked, = click target. 
Trigger item is the item click on which triggers nested popover opening